makefile

```diff
@@ -14,14 +16,18 @@ endef
 
 # CI build
 .PHONY: ci
-ci: gen-java gen-swagger
+ci: gen-go gen-java gen-swagger
 
 # Generate ProtoBuf implementation for Go.
 .PHONY: gen-go
 gen-go:
-	$(foreach file,$(PROTO_FILES),$(call exec-command,protoc --go_out=plugins=grpc:$(GOPATH)/src $(file)))
 	rm -rf ./$(GENDIR)/go
```

Comment:
Also moved the `rm` command to the top, otherwise we delete Go files we just generated, when the source is checked out into GOPATH (see below).